src/main/java/com/footbalmanager/app/controller/TeamController.java

```diff
@@ -0,0 +1,47 @@
+package com.footbalmanager.app.controller;
+
+import com.footbalmanager.app.dto.team.PatchTeamRequestDto;
+import com.footbalmanager.app.dto.team.PostTeamRequestDto;
+import com.footbalmanager.app.services.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin("http://localhost:8080")
+public class TeamController {
+    @Autowired
+    private TeamService teamService;
+
+    @GetMapping("/teams")
+    public ResponseEntity<?> getTeams() {
+        return new ResponseEntity<>(teamService.findAll(), HttpStatus.OK);
+    }
+
+    @GetMapping("/teams/{teamId}")
+    public ResponseEntity<?> getTeam(@PathVariable Long teamId) {
+        return new ResponseEntity<>(teamService.findOne(teamId), HttpStatus.OK);
+    }
+
+    @PostMapping("/teams")
+    public ResponseEntity<?> postTeam(@RequestBody PostTeamRequestDto dto) {
```

Comment:
ta nazwa post team, patch team jakoś mnie nie przekonuje, nie powinno być save/create i update? Czy to jakaś konwencja nazewnicza w springu?

Reply:
Nie, u mnie takiej się trzymaliśmy, mogę zmienić.